docs/contracts/contracts/LSP7DigitalAsset/presets/LSP7CompatibleERC20Mintable.md

```diff
@@ -23,7 +23,7 @@
 constructor(string name_, string symbol_, address newOwner_);
 ```
 
-_Deploying a `LSP7CompatibleERC20Mintable` token contract with: token name = `name_`, token symbol = `symbol_`, and address `newOwner_` as the token contract owner._
+_Deploying a `LSP7CompatibleERC20Mintable` token contract with: token name = `name_`, token symbol = `symbol*`, and address `newOwner*` as the token contract owner.\_
```

Comment:
Something might have gone wrong here. Maybe it's related to the local prettier settings on your machine?

The name of the constructor parameters are `symbol_`, but prettier seems to think that it's to put the text in italic.

We need to find how to prevent that.